### ISSUE TITLE
[🐛 Bug]: Log does not scroll to last line while script running

### ISSUE THREAD

**Issue by fogoat** [SeleniumHQ/selenium-ide#1694]
### What happened?

I place echo statements in my side script to display loop counters or the progress of my script. The log does not always scroll to the very end. I am unable to reproduce consistently or can not explain why it works sometimes, but not other times.

### How can we reproduce the issue?

```shell
I am unable to include a script as it deals with a private web site (financial) that requires a login session before running.
```

### Relevant log output

```shell
n/a
```

### Operating System

Windows 10 Pro Version	22H2 Installed on	‎2/‎17/‎2021 OS build	19045.3391 Experience	Windows Feature Experience Pack 120.2212.3920.0

### Selenium version

Selenium IDE Chrome Version 3.17.2

### What are the browser(s) and version(s) where you see this issue?

Currently using Chrome Version 116.0.5845.96 (Official Build) (64-bit)

### What are the browser driver(s) and version(s) where you see this issue?

?

### Are you using Selenium Grid?

?

**Comment by toddtarsi** [comment]
@sunk818 - I'm sorry, fixing issues like this in v3 really isn't a priority for me anymore. If you encounter something like this in v4, I'd be happy to fix. But stabilization and getting a mainline release out there is my top priority.

**Comment by fogoat** [comment]
Understood. When v4 comes out (is it out? I just saw 3.1.72 in the Chrome store), I will be happy to test it out. Thanks again for all your hard work and making Selenium IDE accessible to non-programmers. For years, I tried to use stuff like iMacros or other "web macro" programs and this one is the best and free.

**Comment by toddtarsi** [comment]
I publish the alpha here if you'd like to try it :) 

https://github.com/SeleniumHQ/selenium-ide/releases/tag/latest

It needs documentation, some stabilizations, and code signing certificates. I should probably announce it to be in beta by now tbh.

**Comment by fogoat** [comment]
I was looking for an updated Chrome extension. The IDE runs an internal browser and I prefer to use my external Chrome browser

**Comment by toddtarsi** [comment]
Yeah, but the problem is that Chrome is deprecating manifest v2. And manifest v3 disables arbitrary code execution. By design, that means functions like 'execute script', 'store', or 'assert' basically can't work. So the only choice for the existing web extension model is either this or to slash half of the commands, many of them extremely essential.

**Comment by toddtarsi** [comment]
Additionally, this was never doing a very good job of testing your browser. For example, the local playback actually used emulated JS instead of driver directives. The reproducability of everything shoots up immensely here. In addition to that, I'm working on making the underlying driver configurable, so you will be able to drive a chrome / firefox / whatever browser with it as long as you have the driver. It's on the six month roadmap for this project. Check back in October if that's the main blocker for you.
